patches/chrome-browser-resources-settings-appearance_page-appearance_fonts_page.html.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/browser/resources/settings/appearance_page/appearance_fonts_page.html b/chrome/browser/resources/settings/appearance_page/appearance_fonts_page.html
+index 30194ed4b0109a74991b35af819302eafd9ab59d..cf6020f29de29da841779d7d96ba919af4c176b4 100644
+--- a/chrome/browser/resources/settings/appearance_page/appearance_fonts_page.html
++++ b/chrome/browser/resources/settings/appearance_page/appearance_fonts_page.html
+@@ -119,6 +119,7 @@
+         $i18n{quickBrownFox}
+       </div>
+     </div>
++<if expr="_google_chrome">
```

Comment:
doesn't this give access to things like allowing extensions in incognito mode? Why do we want to hide it?

Reply:
any reason not to also support that extension?

Reply:
Right now this extension is not supported yet, I'll open a follow up issue for handling this later.

Reply:
follow up issue created in https://github.com/brave/brave-browser/issues/415